### DISCUSSION TITLE
Rankups

### DISCUSSION THREAD

**Discussion by NipNacks** [Zrips/CMI#3541] [Q&A]
**Question: **
Hello!

I recently bought this plugin and have been configuring the configs for quite some time now but I can't seem to find out how to connect the CMI Rankup thing to LuckPerms. Is it possible, and is there a way to do it? I've tried looking it up but I haven't find anything useful to help me with my problem. my server have about 20 total ranks, Peasant lvl 1-5, Knight 1-5, HighGuard 1-5, and King lvl 1-5.... All these ranks are setup on Luckperms and good to go. I literally have no idea how to connect these ranks to CMI ranks.. i wanted it to be EXP based... Please try to get to my issue request as soon as possible!

Waiting for your reply, NipNacks
Thank you!

**Comment by CloudeLecaw** [comment]
To not mix up things. lets talk about Group when we talk about Luckperms and Ranks when we talk about CMI Ranks

- you define all 20 Ranks in CMI + a default starter rank
- Then in the Ranks you define the requierments
- On each rank you use the lp command to get into your permission group

more informaton how to do it you find here
https://www.zrips.net/cmi/rankup-system/
